website/pages/en/resources-showcase.js

```diff
@@ -27,13 +27,16 @@ function getBackgroundForCategory(category) {
 }
 
 function renderImg(img, isSmall, key) {
-  return <img src={img}
-    style={{
-      maxHeight: isSmall ? 40 : 500,
-      minHeight: isSmall ? 40 : 220,
-      minWidth: isSmall ? 40 : 220,
-      verticalAlign: 'end',
-    }} key={key} />;
+  return  <div
+  style={{
+    maxHeight: isSmall ? 40 : 600,
+    minHeight: isSmall ? 40 : 320,
+    minWidth: isSmall ? 40 : 320,
+    verticalAlign: 'baseline',
+  }}
+  ><img src={img}
+ key={key} />
+    </div>;
```

Comment:
At this point you could probably scrap `img` and just set `background-image` on that `div`. Probably.

Reply:
I don't think that does the same thing. Here's what I get if I replace img with ```backgroundImg: `url(${img})` ```

![image](https://user-images.githubusercontent.com/22989529/142276426-4d43973b-9286-4e96-a883-0413f2c76395.png)